src/app/utils/vote.ts

```diff
@@ -11,3 +12,6 @@ const voteFilters: Record<VoteType, VoteFilter> = {
 }
 
 export const getFilterForVoteType = (voteType: VoteType): VoteFilter => voteFilters[voteType]
+
+export const getFilterForVoterNameFragment = (fragment: string | undefined) =>
+  fragment ? (vote: ExtendedVote) => hasTextMatch(vote.validator?.media?.name, [fragment]) : () => true
```

Comment:
Bit difficult to read.

Maybe:

```suggestion
export const getFilterForVoterNameFragment = (fragment: string | undefined) => {
   if(!fragment) { return () => true }
   return (vote: ExtendedVote) => hasTextMatch(vote.validator?.media?.name, [fragment])
 }
```